src/createDijit.ts

```diff
@@ -166,7 +165,7 @@ function resolveCtor<D extends DijitWidget>(Ctor: DijitWidgetConstructor<D> | st
 				handle.remove();
 				reject(error);
 			});
-			const mid = Ctor;
+			const mid = <string> Ctor;
```

Comment:
Hmmm... I am surprised with flow control that this hadn't been narrowed to `string` here.